src/main/java/frc/robot/SpoolIntake.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.Constants;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class SpoolIntake extends Command {
+    private final IntakeSubsystem intakeSubsystem;
+    private final Timer timer = new Timer();
+    private final double motorRunTime; // temporary until  done testing
+
+    public SpoolIntake(IntakeSubsystem intakeSubsystem, double motorRunTime) {
+        addRequirements(intakeSubsystem);
+        this.intakeSubsystem = intakeSubsystem;
+        this.motorRunTime = motorRunTime;
+    }
+
+    @Override
+    public void initialize() {
+        timer.reset();
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        intakeSubsystem.setMotorSpeed(Constants.INTAKE_MOTOR_1_SPEED, Constants.INTAKE_MOTOR_2_SPEED);
+    }
+
+    @Override
+    public boolean isFinished() {
+        return timer.hasElapsed(motorRunTime);
+    }
+}
```

Comment:
We don't stop the intake in the end method. I understand why you're doing this, but the problem is that if this gets interrupted or cancelled, the motor may continue spinning.

Reply:
made wheels only stop if command is interrupted